extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -8,11 +8,31 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 	'ext.wikia.adEngine.provider.gpt.adElement',
 	'ext.wikia.adEngine.provider.gpt.googleTag',
 	'ext.wikia.adEngine.slotTweaker',
+	require.optional('ext.wikia.adEngine.provider.gpt.sourcePointTag'),
 	require.optional('ext.wikia.adEngine.provider.gpt.sraHelper')
-], function (log, adContext, adLogicPageParams, adDetect, AdElement, googleTag, slotTweaker, sraHelper) {
+], function (
+	log,
+	adContext,
+	adLogicPageParams,
+	adDetect,
+	AdElement,
+	GoogleTag,
+	slotTweaker,
+	SourcePointTag,
+	sraHelper
+) {
 	'use strict';
 
-	var logGroup = 'ext.wikia.adEngine.provider.gpt.helper';
+	var logGroup = 'ext.wikia.adEngine.provider.gpt.helper',
+		context = adContext.getContext(),
+		googleApi;
+
+	if (context.opts.sourcePoint) {
+		log('SourcePoint enabled', 'debug', logGroup);
+		googleApi = new SourcePointTag();
+	} else {
+		googleApi = new GoogleTag();
```

Comment:
I'm wondering if additional `log()` (similar to `SourcePoint` one) will hurt us here. What do you think?